latch_cli/services/register/register.py

```diff
@@ -330,6 +334,29 @@ def register(
             click.secho("\n`snakemake` package is not installed.", fg="red", bold=True)
             sys.exit(1)
 
+    workspaces = get_workspaces()
+    ws_name = next(
+        (
+            x[1]
+            for x in workspaces.items()
+            if x[0] == current_workspace()
+            or (current_workspace() == "" and x[1] == "Personal Workspace")
+        ),
+        "N/A",
+    )
+
+    if not skip_workspace_selection:
+        if click.confirm(
+            f"Workflow will be registered to {ws_name} ({current_workspace()})."
```

Comment:
does it makes sense to have both ws_name and account_id
ik its unlikely but possible for ws names to be the same lol